blog/2019-10-08-handling-unchanged-postgres-toast-values.adoc

```diff
@@ -0,0 +1,316 @@
+= Strategies for Handling Unchanged Postgres TOAST Values
+gmorling
+:awestruct-tags: [ discussion, examples, postgres, kafka-streams ]
+:awestruct-layout: blog-post
+
+Let's talk about TOAST.
+Toast?
+No, TOAST!
+
+So what's that?
+https://www.postgresql.org/docs/current/storage-toast.html[TOAST] (The Oversized-Attribute Storage Technique) is a mechanism in Postgres which allows to store large column values in multiple physical rows, circumventing the page size limit of 8 KB.
```

Comment:
allows to store -> stores ... just a suggestion.

Reply:
Changed, thanks.